src/components/bookComponents/BookItem.js

```diff
@@ -1,17 +1,18 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ]  `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.